scripts/build/rollup.config.js

```diff
@@ -63,6 +97,7 @@ export const iife = {
     sourcemap: true,
   },
   plugins: [
+    updateLibPrefixPlugin(),
```

Comment:
I think another possible solution is using https://www.npmjs.com/package/rollup-plugin-inject-process-env, so we can dynamically inject environment variable here, then read in the build process.
Manipulating the library file solution LGTM too.

Reply:
Great solution! I'll make it as a backup if the plugin solution doesn't work well.